app/script/auth/page/CreateAccount.js

```diff
@@ -35,25 +44,24 @@ const CreateAccount = ({history}) => (
           <div>
             <H1 center>{'Set up your account'}</H1>
             <Form>
-              <Input placeholder={'Name'.toUpperCase()} autoFocus />
-              <Input type="email" placeholder={'you@yourcompany.com'} />
-              <Input type="password" placeholder={'Password (min 8 characters)'.toUpperCase()} />
+              <InputBlock>
+                <Input placeholder={'Name'} autoFocus />
+                <Input placeholder={'you@yourcompany.com'} placeholderTextTransform="unset" />
+                <Input placeholder={'Password (min 8 characters)'} />
+              </InputBlock>
               <Checkbox>
                 <Small textTransform="uppercase">
                   {'I ACCEPT THE '}
-                  <Link href="#">{'TERMS AND CONDITIONS'}</Link>
+                  <Link href="#" bold fontSize="12px">
+                    {'TERMS AND CONDITIONS'}
```

Comment:
Let's come up with a system that allows moving this link around. In some languages it is hard because you split a sentence into two and enforce the second part to be the ToUs. Let's put this into a single sentence instead where we replace with the link tag on our own. @lipis